content/en/docs/cluster-administration/platform-settings/notification-management/configure-email.md

```diff
@@ -1,10 +1,62 @@
 weight: 8721
 ---
 
-TBD
+This tutorial demonstrates how to configure your email server and add recipients, who can receive email notifications for alerting policies.
+
+## Configure the Email Server
+
+1. Log in to the web console with an account granted the role  `platform-admin`.
+
+2. Click **Platform** in the top left corner and select **Platform Settings**.
+
+3. Navigate to **Email** under **Notification Management**.
+
+   ![emai-server](/images/docs/cluster-administration/platform-settings/notification-management/configure-email/email-server.png)
+
+4. Under **Server Settings**, configure your email server by filling in the following fields.
+
+   - **SMTP Server Address**: The SMTP server address that can provide email services. The port is usually `25`.
+   - **Use SSL Secure Connection**: SSL can be used to encrypt emails, thereby improving the security of information transmitted by email. Usually you have to configure the certificate for the email server.
+   - **SMTP User**: The SMTP account.
+   - **SMTP Password**: The SMTP account password.
+   - **Sender Email Address**: The sender's email address. Customized email addresses are currently not supported.
+
+5. Click **Save**.
+
+## Add Recipients
+
+1. Under **Recipient Settings**, enter a recipient's email address and click **Add**.
+
+2. After it is added, the email address of a recipient will be listed under **Recipient Settings**. You can add multiple recipients so that all of them will be able to receive email notifications of alerts.
+
+3. To remove a recipient, hover over the email address you want to remove, then click the trash bin icon that appears.
+
+4. To make sure notifications will be sent to your recipients, turn on **Receive Notifications** and click **Update**.
+
+   {{< notice note >}}
+
+   If you change the existing configuration, you must click **Update** to apply it.
+
+   {{</ notice >}} 
+
+## Receive Email Notifications
+
+After you configure the email server and add recipients, you need to enable [KubeSphere Alerting](../../../../pluggable-components/alerting/) and create an alerting policy for workloads or nodes. Once it is triggered, all the recipients can receive email notifications.
+
+The image below is an email notification example:
+
+![example-email-notification](/images/docs/cluster-administration/platform-settings/notification-management/configure-email/example-email-notification.png)
+
+{{< notice note >}}
+
+- If you update your email server configuration, KubeSphere will send email notifications based on the latest configuration.
+- By default, KubeSphere sends notifications for the same alert about every 12 hours. The notification repeat interval is controlled by `group_interval`, `group_wait` and `repeat_interval` in the Secret `alertmanager-main` in the project `kubesphere-monitoring-system`. You can customize these values but keep in mind that the actual time interval may be slightly different.
```

Comment:
Compared with `repeat_interval`, `group_interval` and `group_wait` have little impact on the time of resending the same notification. The official recommended value of `repeat_interval` is more than 3 hours, and `group_interval` is usually in minutes. So I think there is no need to mention `group_interval`, `group_wait` here, this will only increase the difficulty of understanding for users.

Reply:
Agree, they're Alertmanager configs.
And notification manager could receive notification from other sources other than Alertmanager

Reply:
Changed.